### PR TITLE
SAK-31818 - Ensure ARIA roles, states, and properties are valid on Mrphs-sitesNav__menuitem

### PR DESCRIPTION
**Pull request by hunt-son**
SAK-31818 modified where attributes exist in span tag line 23 on includeSitesNav.vm

### CODE REVIEW

**Comment by csev** [comment]
Hunter has an ICLA sent to lithe licensing address.  He is a University of Michigan Computer Science student.

File: portal/portal-render-engine-impl/pack/src/webapp/vm/morpheus/includeSitesNav.vm [hunk 1]
```diff
@@ -20,8 +20,7 @@
                 #if (!${site.isMyWorkspace} && (${site.favorite} == "true" || ${site.isCurrentSite} || !${userIsLoggedIn}))
                     <li class="Mrphs-sitesNav__menuitem #if (${site.isCurrentSite}) is-selected #end">
                         <a href="${site.siteUrl}" title="${site.fullTitle}" role="menuitem" aria-haspopup="true">
-                            <span>#if ( ( ${tabDisplayLabel} == 2 ) && ( ${site.shortDescription} ) )${site.shortDescription}#else${site.siteTitle}#end</span>
-                            <span class="Mrphs-sitesNav__dropdown" tabindex="-1" data-site-id="${site.siteId}"></span>
+                            <span class="Mrphs-sitesNav__dropdown" tabindex="-1" data-site-id="${site.siteId}>#if ( ( ${tabDisplayLabel} == 2 ) && ( ${site.shortDescription} ) )${site.shortDescription}#else${site.siteTitle}#end"</span>
```
Comment (jonespm) [inline portal/portal-render-engine-impl/pack/src/webapp/vm/morpheus/includeSitesNav.vm]: I believe the double quote here is incorrectly placed and should be within the <span> tag after site.siteId} not all the way at the end

**Comment by jonespm** [comment]
Also the message on the commit should contain match up with the jira title, so for this one 

SAK-31818 - Ensure ARIA roles, states, and properties are valid on Mrphs-sitesNav__menuitem

**Comment by ncaidin** [comment]
CLA has been received and is on file.

**Comment by hunt-son** [comment]
Changes were made as requested. It should be good to go now.

**Comment by alejandrogj** [comment]
Hi there!!

This commit created an issue with the sites nav

https://jira.sakaiproject.org/browse/SAK-31886

Regards!

**Comment by ottenhoff** [comment]
This commit is preventing other UI blockers from being addressed. If we can't find a quick fix, we should discuss reverting this commit.

**Comment by SedueRey** [comment]
@ottenhoff I'm taking a look on this regression. I hope I can do it and respect ARIA roles

**Pull request merged**